enabled.vcl

```diff
@@ -117,11 +118,23 @@ sub vcl_hash {
         hash_data(server.ip);
     }
 
+    if (req.url ~ "/graphql") {
+        call process_graphql_headers;
+    }
+
     # To make sure http users don't see ssl warning
     if (req.http.X-Forwarded-Proto) {
         hash_data(req.http.X-Forwarded-Proto);
     }
+}
 
+sub process_graphql_headers {
+    if (req.http.Store) {
+        hash_data(req.http.Store);
+    }
+    if (req.http.Content-Currency) {
+        hash_data(req.http.Content-Currency);
+    }
 }
```

Comment:
@mikeymike this piece is crucial for some of our upcoming work, need to get this change merged ASAP :)